### ISSUE TITLE
Defining wildcard-import exceptions does not seem to work with default config

### ISSUE THREAD

**Issue by sschuberth** [arturbosch/detekt#1468]
## Expected Behavior

Even with `buildUponDefaultConfig = true` , I'd expect the following config to exclude `ch.frankel.slf4k.*` from wildcard-import checks:

```
style:
  WildcardImport:
    excludeImports: '
      ch.frankel.slf4k.*
      '
```

## Observed Behavior

`excludeImports` seems to be ignored with `buildUponDefaultConfig = true`.

## Steps to Reproduce

Checkout branch *detekt-rc-13-bug* of https://github.com/heremaps/oss-review-toolkit and run `./gradlew detekt`. See how `ch.frankel.slf4k.*`  imports are reported as errors.

## Your Environment

* Version of detekt used: 1.0.0-RC13 (with `buildUponDefaultConfig = true`)
* Version of Gradle used (if applicable): 5.2.1
* Operating System and version: Windows 10
* Link to your project (if it's a public repository): https://github.com/heremaps/oss-review-toolkit/tree/detekt-rc-13-bug

**Comment by marschwar** [comment]
Thank you for supplying code to reproduce the issue.

The `WildcardImport` rule works fine. Even in combination with `buildUponDefaultConfig`. What you have been seeing in the output is the `NoWildcardImports` rule which is a ktlint formatting rule. That rule is enabled by default and can (and in your case should) be disabled with:

```
formatting:
  ...
  NoWildcardImports:
    active: false
```

**Comment by sschuberth** [comment]
Oh, that's subtle! Thanks for finding this out. Feel free to close this as invalid then.

**Comment by arturbosch** [comment]
Thanks for the investigation @marschwar !
